modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -661,7 +661,7 @@ export default {
           }
         }
         // New documents only
-        newRelatedDocs = relatedDocs.filter(doc => unlocalizedIds.has(doc._id));
+        relatedDocs = relatedDocs.filter(doc => unlocalizedIds.has(doc._id));
```

Comment:
Unrelated fix; this was producing console turds and breaking the filtering for the "new related documents only" case.